lib/pushmi_pullyu/aip_logger.rb

```diff
@@ -0,0 +1,49 @@
+require 'rdf'
+require 'rdf/n3'
+require 'fileutils'
+require 'pushmi_pullyu/fedora_object_fetcher'
+require 'pushmi_pullyu/solr_fetcher'
+
+class PushmiPullyu::AipLogger
```

Comment:
Yeah I would echo this statement too. Not seeing much benefit for this? Alot of this stuff can be added to the main logging.rb class if required. Preservation event as well.